diagnostic.md

```diff
@@ -157,7 +192,9 @@ div {
 ```
 
 <!-- your answer starts here -->
-
+divOne will float to the left corner of the screen and take along divTwo and
+divThree to its right.  divFour will stay inline below divOne because it was
+cleared of the float.
```

Comment:
Close, .divOne will be in the top-right corner; .divTwo will be in the top-left corner, with .divThree immediately to its right; .divFour will be underneath .divTwo, and .divFive will sit immediately to the right of .divFour (under .divThree).